packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -572,14 +593,37 @@ export class ExternalRequest {
       if (!linkTable || !linkPrimary) {
         return
       }
+
+      // @ts-ignore
```

Comment:
Sorry just noticed this. Best not to use a ts-ignore. (It's caused issues before!)

Reply:
I just added this to relace the old `if(primary &...)`. I don't like using the @ts-ignore, but this otherwise we need either the full check or disabling this https://www.typescriptlang.org/tsconfig#noUncheckedIndexedAccess.
In javascript `['a','b'][4]` will return undefined, so this is what we would expect. This `@ts-ignore` pattern is being used multiple times in this file, I just replicate it to match this use case. This is a problem with the typings that would need a better understanding of these keys. I don't think we can have a relationship without the primary, but currently the types support it :(

Reply:
Anyways, I just cleaned the one I introduced